client/src/leap/soledad/client/http_target/fetch.py

```diff
@@ -248,4 +248,5 @@ def _emit_receive_status(received_docs, total):
     emit_async(SOLEDAD_SYNC_RECEIVE_STATUS, content)
 
     msg = "%d/%d" % (received_docs, total)
```

Comment:
I will prefer to move the setting of msg to the if block to have it together with the log.